Anlab.Mvc/Views/Lab/Orders.cshtml

```diff
@@ -17,11 +17,11 @@
             <td>
                 <select name="status" id="status">
                     <option value="All">Show All</option>
-                    <option value="Confirmed">Confirmed</option>
-                    <option value="Received">Received</option>
-                    <option value="Finalized">Finalized</option>
-                    <option value="Accepted">Accepted</option>
-                    <option value="Complete">Complete</option>
+                    <option value="@OrderStatusCodes.Confirmed">@OrderStatusCodes.Confirmed</option>
+                    <option value="@OrderStatusCodes.Received">@OrderStatusCodes.Received</option>
+                    <option value="@OrderStatusCodes.Finalized">@OrderStatusCodes.Finalized</option>
+                    <option value="@OrderStatusCodes.Accepted">@OrderStatusCodes.Accepted</option>
+                    <option value="@OrderStatusCodes.Complete">@OrderStatusCodes.Complete</option>
```

Comment:
@jSylvestre do you remember how we do this in GREAT?  I'm thinking make we throw the OrderStatusCodes into an array and then do a foreach?  Just want to future proof for the case where we add another status so we don't have to change this